swift-test-project/Sources/app/main.swift

```diff
@@ -55,4 +55,12 @@ if let result = client.post(
       print(org as Any)
     }
   }
+case .failure(let error):
+  print("""
+        [Error] \(error.localizedDescription)
```

Comment:
To make the message more generic, I chose to output the given error message.

It would be like the following in case the server not running.

```
[Error] Could not connect to the server.
To run the server, see README:
https://github.com/mtsmfm/graphql-codegen-swift-operations/tree/master/swift-test-project
```